src/wireviz/Harness.py

```diff
@@ -110,14 +111,21 @@ def create_graph(self) -> Graph:
                 pinhtml = []
                 pinhtml.append('<table border="0" cellspacing="0" cellpadding="3" cellborder="1">')
 
-                for pin, pinlabel in zip(connector.pins, connector.pinlabels):
+                for pin, pinlabel, pincolor in zip_longest(connector.pins, connector.pinlabels, connector.pincolors):
                     if connector.hide_disconnected_pins and not connector.visible_pins.get(pin, False):
                         continue
                     pinhtml.append('   <tr>')
                     if connector.ports_left:
                         pinhtml.append(f'    <td port="p{pin}l">{pin}</td>')
                     if pinlabel:
                         pinhtml.append(f'    <td>{pinlabel}</td>')
+                    if connector.pincolors:
+                        if pincolor in wv_colors._color_hex.keys():
+                            pinhtml.append(f'    <td sides="tbl">{pincolor}</td>')
+                            pinhtml.append(f'    <td sides="tbr"><table border="0" cellborder="1"><tr><td bgcolor="{wv_colors.translate_color(pincolor, "HEX")}" width="8" height="8" fixedsize="true"></td></tr></table></td>')
+                        else:
+                            pinhtml.append(f'    <td colspan="2"></td>')
```

Comment:
An f-string is not needed here, but if you want it for consistency, it doesn't really harm...
```suggestion
                            pinhtml.append('    <td colspan="2"></td>')
```